sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitImporter.java

```diff
@@ -23,9 +23,8 @@
 import java.util.Map;
 import org.sonar.api.batch.sensor.SensorContext;
 
-@FunctionalInterface
-public interface PhpUnitParser {
+public interface PhpUnitImporter {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Annotate the "PhpUnitImporter" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1609)